packages/expo-image-picker/src/ImagePicker.ts

```diff
@@ -11,6 +11,26 @@ import {
   VideoExportPreset,
 } from './ImagePicker.types';
 
+function validateOptions(options: ImagePickerOptions) {
+  const { aspect, quality, videoMaxDuration } = options;
```

Comment:
```suggestion
function validateOptions({ aspect, quality, videoMaxDuration }: ImagePickerOptions) {
```

Reply:
If I do it this way, I won't be able to return `options` object, and thus inline `validateOptions()` as you suggested